lib/Logger/LoggerFeature.h

```diff
@@ -79,6 +82,8 @@ class LoggerFeature final : public application_features::ApplicationFeature {
   bool _supervisor = false;
   bool _backgrounded = false;
   bool _threaded = false;
+  std::string _apiSwitch;
+  bool _apiEnabled;
```

Comment:
The other bool switches are explicitly set.

Reply:
True, I have used the old-fashioned way to initialize in the Constructor. Should I change this? I would have to do this in 5 PRs and run Jenkins 5 more times. I can do this, if we think it is worthwhile...